src/components/emailerFilterControls/index.tsx

```diff
@@ -0,0 +1,235 @@
+    (
+      key: 'neighborhoods' | 'commonNames',
+      options: { value: string }[],
+      searchValue: string,
+      warning: string,
+    ) => {
+      return () => {
+        if (options.map((option) => option.value).includes(searchValue)) {
+          if (!filters[key].includes(searchValue)) {
+            setFilters({ ...filters, [key]: [...filters[key], searchValue] });
+            key === 'neighborhoods'
+              ? setNeighborhoodSearch('')
+              : setNamesSearch('');
+          }
+        } else {
+          message.warn(warning);
+        }
+      };
+    },
+    [filters, setFilters],
+  );
+
+  return (
+    <StyledCollapse ghost={true}>
+      <Collapse.Panel header="Activity Count" key="activityCount">
+        <p>{`${filters.activityCountMin} - ${
+          filters.activityCountMax || MAX_COUNT + '+'
+        }`}</p>
+        <Slider
+          range={true}
+          marks={activityCountLabels}
+          tooltip={{ open: false }}
+          min={0}
+          max={MAX_COUNT + 1}
+          value={activityCountRange(filters)}
+          onChange={([min, max]) => {
+            setFilters({
+              ...filters,
+              activityCountMin: min,
+              activityCountMax: max > MAX_COUNT ? undefined : max,
+            });
+          }}
+        />
+      </Collapse.Panel>
+      <Collapse.Panel header="Adoption Date" key="adoptionDate">
+        <StyledRangePicker
+          value={formatDates(filters.adoptedStart, filters.adoptedEnd)}
+          onChange={(_, dateStrings) =>
+            setFilters({
+              ...filters,
+              adoptedStart: dateStrings[0],
+              adoptedEnd: dateStrings[1],
+            })
+          }
+          disabledDate={disabledDate}
+        />
+        <Button
+          type="primary"
+          onClick={() =>
+            setFilters({
+              ...filters,
+              adoptedStart: undefined,
+              adoptedEnd: undefined,
+            })
+          }
+        >
+          <CloseOutlined />
+        </Button>
+      </Collapse.Panel>
+      <Collapse.Panel header="Last Activity Date" key="lastActivityDate">
+        <StyledRangePicker
+          value={formatDates(
+            filters.lastActivityStart,
+            filters.lastActivityEnd,
+          )}
+          onChange={(_, dateStrings) =>
+            setFilters({
+              ...filters,
+              lastActivityStart: dateStrings[0],
+              lastActivityEnd: dateStrings[1],
+            })
+          }
+          disabledDate={disabledDate}
+        />
+        <Button
+          type="primary"
+          onClick={() =>
+            setFilters({
+              ...filters,
+              lastActivityStart: undefined,
+              lastActivityEnd: undefined,
+            })
+          }
+        >
+          <CloseOutlined />
+        </Button>
+      </Collapse.Panel>
+      <Collapse.Panel header="Neighborhood" key="neighborhood">
+        <AutoComplete
```

Comment:
🔧 Instead of having the user choose and add the neighborhood or common name (and have it show up below the filters), maybe it might be easier if we use [multiple select](https://ant.design/components/select#components-select-demo-multiple) instead?

Reply:
Yeah that looks way better - I put the filters at the bottom as a placeholder since I think they're supposed to go above the table (which isn't there yet), but having them right in the search bar would probably be easier